packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/routing.js

```diff
@@ -8,11 +8,27 @@ export default /* @ngInject */ ($stateProvider) => {
     },
     resolve: {
       nodeId: /* @ngInject */ ($transition$) => $transition$.params().nodeId,
-      orderPrivateBandwidthLink: /* @ngInject */ ($state, nodeId) =>
+      handleError: /* @ngInject */ (Alerter) => (error) => {
+        Alerter.alertFromSWS(error.message, 'ERROR', 'nutanix_node_alert');
+      },
+      goBack: /* @ngInject */ ($state, $timeout) => (scrollToTop) => {
+        $state.go('^');
+
+        if (scrollToTop) {
+          $timeout(() => {
+            const element = document.getElementById('nutanix_dashboard_node');
+            if (element) {
+              element.scrollIntoView({ behavior: 'smooth', block: 'start' });
+            }
+          }, 300);
+        }
+      },
```

Comment:
What about `$anchorScroll` instead ?

Reply:
I tried and it didn't work : with location.hash and $anchorScroll the modal was reopened